app/assemblers/assemble_representation_context.rb

```diff
@@ -42,7 +42,7 @@ def representation_count
     def supermentored_tracks
       return Track.all if mentor.staff?
 
-      Track.where(id: mentor.track_mentorships.supermentor_frequency.select(:track_id)).order(title: :asc)
+      Track.where(id: mentor.track_mentorships.automator.select(:track_id)).order(title: :asc)
```

Comment:
We no longer use the scope that checked for frequency, but use the new scope that is the flag instead. You'll see this throughout.